CRM/Member/BAO/Membership.php

```diff
@@ -1876,11 +1853,13 @@ public static function processMembership($contactID, $membershipTypeID, $is_test
           'join_date' => $currentMembership['join_date'],
           'membership_type_id' => $membershipTypeID,
           'max_related' => !empty($membershipTypeDetails['max_related']) ? $membershipTypeDetails['max_related'] : NULL,
+          'is_pay_later' => $isPayLater,
```

Comment:
per comments above - let's not propagate this parameter - if we need to control the membership_activity_status_id let's call it that.